server/weaverbird/steps/concatenate.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+
+from pandas import DataFrame
+from pydantic import Field
+
+from weaverbird.steps.base import BaseStep
+
+ColumnName = str
```

Comment:
May we move than to a shared place to avoid having redefining that for all steps?